### DISCUSSION TITLE
tls/ssl(PORT 443) loses about 6% of connections, but tcp(port 1883) loses barely 0.2%

### DISCUSSION THREAD

**Discussion by springmvcGitHub** [rabbitmq/rabbitmq-server#9007] [Other]
Hi expert:
Now with 53,000 client connections to test, tls/ssl(port 443) connection loss rate of about 6%, while tcp(port 1883) connection loss rate is very low about 0.2%, whether it is k8s Intranet, or direct connection ip address, or domain name, tls/ssl loss rate is maintained at about 6%. 

Could you help analyze the cause of this problem? I don't know if this phenomenon is related to rtt.

conf:
rabbitmq server:4core 12Gi 3pod
client1:emqtt-bench-tool 4core 8Gi 1pod
client2:emqtt-bench-tool 4core 8Gi 1pod
client3:emqtt-bench-tool 4core 8Gi 1pod

scenario:
client1 Initiates 11,000 connections, 10 data per second, 3k data per second, and sends 30 minutes
client2 initiates 21,000 connections, sends one data piece in 24 seconds, each data piece is 1k, and sends 30 minutes
client2 initiates 21,000 connections, sends one data piece in 8 seconds, each data piece is 1k, and sends 30 minutes

rabbitmq-server config:
      vm_memory_high_watermark.relative = 0.9
      mqtt.tcp_listen_options.sndbuf = 1024
      mqtt.tcp_listen_options.recbuf = 1024
      mqtt.tcp_listen_options.buffer = 1024
      mqtt.tcp_listen_options.backlog = 512
      mqtt.tcp_listen_options.nodelay = true
      vm_memory_high_watermark.relative = 0.95
      vm_memory_high_watermark_paging_ratio = 0.95
      disk_free_limit.absolute = 200MB
      classic_queue.default_version = 2
      management_agent.disable_metrics_collector = true
      log.console = true
      log.console.level = warning
      collect_statistics_interval = 20000
      classic_queue.default_version = 2
      ERL_MAX_PORTS=5000000

tls/ssl (port 443):
<img width="1375" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/42227896/b4bbfc93-840e-46a9-92d1-be1b213d7748">

tcp (port 1883):
<img width="1364" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/42227896/424db63c-3174-4b3c-9e1b-0425b6a901bb">

**Comment by michaelklishin** [comment]
Our team is not available for troubleshooting and RCA of your networking infrastructure. 

Please start with inspecting logs from all nodes for clues, and a traffic capture to see what connections are closed. In the specific case of MQTT, duplicate client IDs is one reasons where nodes must disconnect existing clients.

RabbitMQ logs all [key connection lifecycle events](https://rabbitmq.com/logging.html#logged-events).
